app/lib/handlebars_helpers/identifiers.js

```diff
@@ -0,0 +1,14 @@
+export default {
+  'wdt:P213': id => { return `https://isni.oclc.org/xslt/DB=1.2//CMD?ACT=SRCH&IKT=8006&TRM=ISN%3A${id}` },
```

Comment:
syntactic preference would be to use the arrow function implict return for the beauty of that simpler one-liner
```suggestion
  'wdt:P213': id => `https://isni.oclc.org/xslt/DB=1.2//CMD?ACT=SRCH&IKT=8006&TRM=ISN%3A${id}`,
```

Reply:
yeah i know, but somehow webpack doesnt like it and fails to compile...

Reply:
what's webpack error message?

Reply:
done 884c4b4